store/postgres/tests/store.rs

```diff
@@ -1544,5 +1632,127 @@ fn revert_block_with_partial_update() {
         store.revert_events(revert_event_source);
         let reverted_entity = store.get(entity_key).unwrap();
         assert_eq!(reverted_entity, original_entity);
+
+        Ok(())
+    })
+}
+
+#[test]
+fn entity_changes_are_fired_and_forwarded_to_subscriptions() {
+    run_test(|| {
+        let logger = Logger::root(slog::Discard, o!());
+        let url = postgres_test_url();
+        let mut store =
+            DieselStore::new_without_processing_entity_changes(StoreConfig { url }, &logger);
+
+        // Create a store subscription
+        let subscription = store.subscribe(String::from("subgraph-id"), vec![String::from("User")]);
+
+        // Start processing entity changes from the DB now that we're subscribed
+        store.start_processing_entity_changes();
```

Comment:
Thinking again about this, it no longer makes sense to me. It shouldn't matter that the listener is already started, it should only matter that the store changes are done after the subscription.

Reply:
That is... true. Let me try without this.